files/es/web/javascript/a_re-introduction_to_javascript/index.html

```diff
@@ -317,7 +317,7 @@ <h2 id="Estructuras_de_control">Estructuras de control</h2>
 }
 </pre>
 
-<p>Los operadores <code>&amp;&amp;</code> y <code>||</code> utilizan lógica de cortocircuito, lo cual significa que si ejecutarán su segundo operando depende del primero. Esto es útil para verificar objetos nulos antes de acceder a sus atributos:</p>
+<p>Los operadores <code>&amp;&amp;</code> y <code>||</code> utilizan evaluación de cortocircuito, lo cual significa que la evaluación de su segundo operando depende del valor del primero. Esto es útil para verificar objetos nulos antes de acceder a sus atributos:</p>
```

Comment:
Creo que asi queda mejor. 
@marcelozarate Que opinas?


```suggestion
<p>Los operadores <code>&amp;&amp;</code> y <code>||</code> utilizan evaluación de cortocircuito, lo cual significa que la evaluación del segundo operando depende del valor del primero. Esto es útil para verificar objetos nulos antes de acceder a sus atributos:</p>
```

Reply:
Este cambio arregla completamente este https://github.com/mdn/translated-content/issues/4559#issuecomment-1151690306 issue?

Reply:
Sí, arregla el issue por ahora, lo dejo en html para saber que todavía falta revisar la traducción completa y pasarlo a markdown.